src/task7/app.ts

```diff
@@ -0,0 +1,30 @@
+import mongoose from 'mongoose';
+import { addInitialUsers } from './user/user.service';
+import { addInitialProducts } from './product/product.service';
+import { addInitialCarts } from './cart/cart.service';
+import { addInitialOrders } from './order/order.service';
+
+const uri: string = 'mongodb://localhost:27017';
+
+mongoose
+  .connect(uri, {
+    user: 'root',
```

Comment:
It's better to use `.env` file to store all secret credentials for security reason.